packages/compodoc/src/executors/build/executor.ts

```diff
@@ -20,10 +20,17 @@ export default async function runExecutor(
   options: BuildExecutorSchema,
   context: ExecutorContext,
 ) {
-  options.debug && console.log('Prepare Compodoc...\n', options);
 
   const project = context.workspace.projects[context.projectName];
 
+  options = {
```

Comment:
I would like to know how the options should be accessed by nx.
The reference to the original options object is broken and the executor doesn't return the options 🤔

Reply:
That was the question I was thinking about in https://github.com/twittwer/nx-tools/issues/57#issuecomment-996875091 as well.

Maybe it is required to mutate the options in the options object to keep the reference, something like:
`òptions.outputPath ??= 'default/path'`

Reply:
Ok I understand. Fixed then.

Probably Nx Is wrapping your executor and used the options initially specified. But if they make a copy of it, then this will not work either, the only way will be to make tsConfig and ouputPath mandatory